libcodechecker/libhandlers/analyze.py

```diff
@@ -174,6 +174,14 @@ def is_ctu_capable():
                                     "compiler on the system is special, e.g. "
                                     "when doing cross-compilation.")
 
+    analyzer_opts.add_argument('--capture-analysis-output',
```

Comment:
I wonder, mayhaps _capture-**successful**-output_ could be a better optstring? Considering that failure output is also captured (about which I wonder perhaps we should introduce an optstring that _disables_ that notion, as there could be cases someone does not truly care about the analysis failures as deeply as the code currently does.)